### PR TITLE
fix(issue#14): add new JS question to full-quiz file about Set object

### PR DESCRIPTION
**Pull request by a-cole55**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/full-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,15 @@
 const fullQuiz = [
+  {
+    Question:
+      "In Javascript, which built-in method/object will return the unique values of an array?",
```
Comment (jdwilkin4) [inline src/data/full-quiz.ts:4]: Make sure to use the correct capitalization here for JavaScript

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Hi @a-cole55 ! 

I reverted back to your original commit to remove the changes to the `package.json` file. 
I left one small comment below. 👍

**Comment by jdwilkin4** [comment]
Since I made changes to your copy of the developer quiz site, make sure to pull down those new changes locally.

**Comment by a-cole55** [comment]
I have made the requested change.

**Review by jdwilkin4 — APPROVED** [review]
LGTM! 👍

**Comment by jdwilkin4** [comment]
Congrats @a-cole55 on your first contribution merged into the Developer Quiz Site! 
I look forward to future contributions 👍

**Pull request merged**